ale/formatters/usgscsm_formatter.py

```diff
@@ -0,0 +1,91 @@
+import json
+from ale.base.type_sensor import LineScanner, Framer
+
+def to_usgscsm(driver):
+    isd_data = {}
+
+    # exterior orientation
+    body_radii = driver.target_body_radii
+    isd_data['radii'] = {
+        'semimajor' : body_radii[0],
+        'semiminor' : body_radii[1],
+        'unit' : 'm'
+    }
+    positions, velocities, position_times = driver.positions
+    isd_data['sensor_position'] = {
+        'positions' : positions,
+        'velocities' : velocities,
+        'unit' : 'm'
+    }
+    sun_positions, sun_velocities, _ = driver.sun_positions
+    isd_data['sun_position'] = {
+        'positions' : sun_positions,
+        'velocities' : sun_velocities,
```

Comment:
Do these not need to be "sun_positions" or something similar, or am I misunderstanding?

Reply:
The sun_position object has the same format as the sensor_position object. Why the positions are for is denoted by what object they are a part of.